services/app/src/Style.scss

```diff
@@ -227,13 +227,55 @@ body {
     flex: 8;
     text-align: center;
 
+    .group {
+      padding-left: size(base);
+      padding-right: size(base);
+    }
+
     svg {
       vertical-align: middle;
       padding: 0 size(base);
     }
   }
 }
 
+.player1 {
+  fill: #c4232f; /* magic red */
+}
+.player2 {
+  fill: #00a3bb; /* melee blue */
```

Comment:
I believe these are already available in the shared css constants... if not, we should definitely move these to there and import them :) esp since we reference them each twice